org.dharkael.Flameshot.appdata.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You are not using the file in the manifest, so I assume upstream's one is used. Please contribute your changes there, if any.